_assets/js/breakpoints.js

```diff
@@ -55,7 +56,7 @@
     // media query change
     // ------------------
     function breakpoint(mq) {
-      if (mq.matches) {
+      if (typeof mq.matches !== 'undefined') {
```

Comment:
This is the only place I see `breakpoint()` ever called:

https://github.com/mozilla/extension-workshop/blob/aa3b52edd9b8189972c4ff3202bd7fbcca1764d6/_assets/js/breakpoints.js#L51

Where does it get called with `{ matches: false }`?

Reply:
Those happen based on the event added here: https://github.com/mozilla/extension-workshop/blob/b33a10fe2d04de8fe49ab52b3a8541f07addc7d3/_assets/js/breakpoints.js#L47

`breakpoint` gets called with change events as a result of that.